boot/zephyr/targets/nrf52840_pca10056.h

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (C) 2017, Linaro Ltd
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_TARGETS_NRF52840_PCA10056_
+#define H_TARGETS_NRF52840_PCA10056_
+
+#define FLASH_DRIVER_NAME		CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_ALIGN			4
+#define FLASH_AREA_IMAGE_0_OFFSET	0x08000
+#define FLASH_AREA_IMAGE_0_SIZE		0x7a000
+#define FLASH_AREA_IMAGE_1_OFFSET	0x82000
+#define FLASH_AREA_IMAGE_1_SIZE		0x7a000
+#define FLASH_AREA_IMAGE_SCRATCH_OFFSET	0xfc000
+#define FLASH_AREA_IMAGE_SCRATCH_SIZE	0x01000
```

Comment:
Given how large the image areas are, @mike-scott brought up that we could make a better tradeoff by increasing the scratch size and shrinking the images. @d3zd3z, any suggestions on what values would tune this well?